torch/onnx/symbolic_opset11.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
```

Comment:
Are these two lines needed?

Reply:
In my comments of normal() method, there are some special characters like 'σ' and 'μ' which was used to explain why we implement this function in this way.

I believe these characters caused a new flake8 issue(encoding) which is resolved by including these 2 lines.